src/scss/6-elements/text-tokens/_article-size.scss

```diff
@@ -0,0 +1,8 @@
+@use '../../abstract' as *;
+
+@media (min-width: 1280px) and (max-width: 1330px) {
+  .#{$p}-reduced-article-size {
+    /* original/default is 41.5rem */
+    max-inline-size: 40.5rem !important;
+  }
```

Comment:
can we put this where `  .#{$p}-reduced-article-size {` is defined in the first place, I dont think we need a new file for it 👍🏻